super_dpf.py

```diff
@@ -222,6 +225,56 @@ def sync(self, verify_local):
         if not cloud_filelist:
             logging.info('{} has nothing to sync.'.format(self.service_name))
 
+        logging.info('Done syncing {}'.format(self.service_name))
+
+
+class UnsplashResource(BaseDPF):
+    REQUIRED_SETTINGS = [
+        SettingsItem('preferred_res', 'Preferred Resolution',
+                     help_text='eg 1600x900'),
+        SettingsItem('category', 'Unsplash Category',
+                     help_text='choices: buildings, food, nature, people,'
```

Comment:
Perhaps SettingsItem should be extended to have a choices= parameter list for validation purposes.